.vscode/tasks.json

```diff
@@ -0,0 +1,20 @@
+// See https://go.microsoft.com/fwlink/?LinkId=733558
+// for the documentation about the tasks.json format
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "type": "npm",
+            "script": "watch",
+            "problemMatcher": "$tsc-watch",
+            "isBackground": true,
+            "presentation": {
+                "reveal": "never"
+            },
```

Comment:
<!--
  3 failure:  " should be ' lin..., Unnecessarily quo..., tab indentation e...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: .vscode/tasks.json;
  Line: 13;
-->
- :no_entry_sign: " should be ' line - tasks.json (quotemark)
- :no_entry_sign: Unnecessarily quoted property 'group' found. line - tasks.json (object-literal-key-quotes)
- :no_entry_sign: tab indentation expected line - tasks.json (indent)